src/screens/Login.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  Image,
+  Pressable,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import React, { useState } from 'react';
+import AppButton from '../components/AppButton';
+import CheckBox from '@react-native-community/checkbox';
+
+const Login = () => {
+  const [secureTextEntry, setSecureTextEntry] = useState(true);
+  const [focusEmail, setFocusEmail] = useState(false);
+  const [focusPassword, setFocusPassword] = useState(false);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <StatusBar barStyle={'light-content'} backgroundColor={'#222222'} />
+      <View style={styles.content}>
+        <View>
+          <Text style={styles.title}>
+            Entrar <Text style={styles.highlight}>M-PAY</Text>
+          </Text>
+        </View>
+        <View>
+          <View style={styles.marginView}>
+            <TextInput
+              placeholder="Email"
+              placeholderTextColor="#9F9F9F"
+              onFocus={() => setFocusEmail(true)}
+              onBlur={() => setFocusEmail(false)}
+              style={focusEmail ? styles.inputBoxOnFocus : styles.inputBox}
```

Comment:
...assim fica mais simples ao ler o código aqui
```suggestion
              style={isEmailFocused ? styles.inputBoxOnFocus : styles.inputBox}
```